images/cluster-nfd-operator.yml

```diff
@@ -31,3 +31,9 @@ update-csv:
   manifests-dir: manifests/
   registry: image-registry.openshift-image-registry.svc:5000
   valid-subscription-label: '["OpenShift Kubernetes Engine", "OpenShift Container Platform", "OpenShift Platform Plus"]'
+  image-references-path: manifests/image-references
+image_has_bundle: true
```

Comment:
Think the existence of th key `update-csv` can be taken as the hint that the image has a bundle. 

Even better, starting in 4.15? 4.16? we stated to have `distgit.bundle_component`. Effort is best spent by backporting that to all relevant versions, and make `distgit.bundle_component` a requirement when `update-csv` exists in `validator`.